glide.lock

```diff
@@ -365,10 +363,12 @@ imports:
   - util/strutil
   - util/testutil
 - name: github.com/prometheus/tsdb
-  version: 706602daed1487f7849990678b4ece4599745905
+  version: 2aae939d6cbebac9e4ad7324287bb9a6bd135d13
```

Comment:
mind reverting all the changes to glide.lock except m3x

Reply:
sure